packages/anoncreds/src/services/AnonCredsVerifierServiceOptions.ts

```diff
@@ -0,0 +1,53 @@
+import type { AnonCredsProof, AnonCredsProofRequest } from '../models/exchange'
+import type {
+  AnonCredsCredentialDefinition,
+  AnonCredsRevocationList,
+  AnonCredsRevocationRegistryDefinition,
+  AnonCredsSchema,
+} from '../models/vdr'
+
+export interface VerifyProofOptions {
+  proofRequest: AnonCredsProofRequest
+  proof: AnonCredsProof
+  schemas: {
+    [schemaId: string]: AnonCredsSchema
+  }
+  credentialDefinitions: {
+    [credentialDefinitionId: string]: AnonCredsCredentialDefinition
+  }
+  revocationStates: {
+    [revocationRegistryDefinitionId: string]: {
+      definition: AnonCredsRevocationRegistryDefinition
+      // NOTE: the verifier only needs the accumulator, not the whole state of the revocation registry
+      // Requiring this to be the full state means we need to retrieve the full state from the ledger
+      // as a verifier. This is just following the data models from the AnonCreds spec, but for e.g. indy
+      // this means we need to retrieve _ALL_ deltas from the ledger to verify a proof. While currently we
+      // only need to fetch the registry.
+      revocationLists: {
+        [timestamp: string]: AnonCredsRevocationList
+      }
+    }
+  }
+}
+
+export type VerifyProofMetadata = {
+  schemas: {
+    [key: string]: {
+      ver: string
```

Comment:
"ver" is always "1.0" for schema, revocationRegistryDefinition and credentialDefinition. Can we remove them and just hardcode them in the indy-sdk anoncreds interfaces?